src/servers/graphql-server.ts

```diff
@@ -150,6 +151,7 @@ export const startApolloServer = async ({
             geetest,
             account,
             ip,
+            authorizationService: await CasbinService(),
```

Comment:
Curious why we chose to initialize here instead of within use-case methods like we do with other services

Reply:
To re-use the database connection. This now creates 1 connection per request. Better would be to use a connection pool but I haven't seen a simple way to pass a pre-existing connection to the casbin libraries. Might investigate this when switching to postgres.

Either way 1 instance (and connection) per domain function would be too much overhead. (Note the repositories re-use the connection that is created when mongoose is initialized)